content/en/docs/architecture/release-gating.md

```diff
@@ -34,6 +35,11 @@ Poor candidates include:
 OpenShift CI creates thousands of clusters a week - we can certainly afford to run a few more tests.  However, we want to spread that investment to derive the maximum benefit for everyone, so having OpenShift CI run 30 variations of your exact same tests in slightly different configurations 4 times a day may not be the best way to test.
 {{< /alert >}}
 
+### A new approach to release gating
```

Comment:
nit: Aggregated jobs will not always be new.  How about `### Aggregated jobs` so we have a stable title and link anchor that will last longer? :)

Reply:
agreed. put this in present tense since it's in place now, working and soon will not even be considered "new"

Reply:
Yeah, the case for having a stable link makes a ton of sense.  Updating now.